services/ext/api.go

```diff
@@ -1808,10 +1808,29 @@ func (api *PublicAPI) SetLogLevel(request *requests.SetLogLevel) error {
 	return api.service.messenger.SetLogLevel(request)
 }
 
+func (api *PublicAPI) SetLogNamespaces(request *requests.SetLogNamespaces) error {
+	return api.service.messenger.SetLogNamespaces(request)
+}
+
 func (api *PublicAPI) SetMaxLogBackups(request *requests.SetMaxLogBackups) error {
 	return api.service.messenger.SetMaxLogBackups(request)
 }
 
+func (api *PublicAPI) LogTest() error {
```

Comment:
I couldn't find a more elegant solution. Initially, I reused logs that were already being printed during the flow, but this approach didn't cover all cases and made potential changes to the logs somewhat inflexible.